monitor.py

```diff
@@ -471,6 +472,39 @@ def findpid():
                 return res['pid']
     return None
 
+@app.route('/login', methods=['GET'])
+def get_login():
+    """Get last login details.
+
+    Returns:
+        TYPE: JSON
+    """
+    try:
+        data = []
+
+        details_regex = r"([a-zA-Z]+\s[a-zA-Z]+\s+[0-9]+\s[0-9]+:[0-9]+)\s+(.*)"
+        username_regex = r"^[a-zA-Z0-9]+\s"
+
+        output = subprocess.check_output("last")
```

Comment:
@rejahrehim @adeyosemanputra Using subprocess is the only best option here. Since, `last` command is valid for all flavours and distros of Linux, be it Debian (Ubuntu, Kali...), Fedora etc.

We can parse log file to collect the details, like we are doing it in **System Log Monitor**, but that would not be generic! It would be very OS dependent as log files have very different locations in all the distros. Going with `last` will keep on the safe side, i.e. being generic.